{"name": "Test login attempt with wrong credentials", "status": "passed", "steps": [{"name": "test_login_incorrect", "status": "passed", "attachments": [{"name": "screen", "source": "af1a95b7-621c-49ac-95ad-d76d4db8282b-attachment.png", "type": "image/png"}], "parameters": [{"name": "login_data", "value": "('', '')"}, {"name": "driver", "value": "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"c5fd9ab6d2ec93153bc2845c9a0790c8\")>"}], "start": 1537515532799, "stop": 1537515548050}], "parameters": [{"name": "login_data", "value": "('', '')"}], "start": 1537515532799, "stop": 1537515548050, "uuid": "edf9a579-856c-4415-a967-9217ceb95fd0", "historyId": "f4349e5ae2ba336cc5543edae9a81784", "fullName": "src.tests.test_selenium.TestLogin#test_login_incorrect[login_data2]", "labels": [{"name": "feature", "value": "User management"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "UI"}, {"name": "story", "value": "Login"}, {"name": "parentSuite", "value": "src.tests"}, {"name": "suite", "value": "test_selenium"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "odp1210013"}, {"name": "thread", "value": "20216-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.test_selenium"}]}